docs/fides/docs/quickstart/local_standalone.md

```diff
@@ -41,6 +41,26 @@ fidesctl --version
 fidesctl, version 1.0.0
```

Comment:
nit for sure but more of a follow on thought, wishing there was a way to populate the version with the docs version being viewed 🤔

Reply:
yeah in general i'd like to make our docs "smarter", but thats a ticker for later